docs/Tutorial.md

```diff
@@ -0,0 +1,538 @@
+# LimeApp's Developer Tutorial
+Wellcome, this the developer tutorial for LimeApp developers!
+
+## 1 What do I need to know to collaborate in the LimeApp?
+
+The LimeApp is written in Javascript, HTML and css.
+Having a basic knowledge of these languages will help you to collaborate in the  in the development. You can follow these resources to learn about these languages:
+- Javascript tutorial: https://javascript.info/
+	- We particularly recommend you to read about [Promises](https://es.javascript.info/promise-basics)
+ 
+You can also learn by doing, following this guide and encouraging you to try making changes or adding new screens.
+
+## 2 Architectural Design
```

Comment:
I would remove the manual numbering of the titles... i feel it gets in the way ... what do you think?

Reply:
Yes, github renders a table of content that helps with ordering anyway